pages/community/tsc.js

```diff
@@ -72,6 +74,11 @@ export default function TSC() {
               can also build a great AsyncAPI-based project that we don't have
               yet in our GitHub organization and donate it (we'll ask you to
               stay as a maintainer).
+              Follow this
+              <TextLink href="https://github.com/asyncapi/community/blob/master/TSC_MEMBERSHIP.md" target="_blank" className={twMerge(`text-base font-normal text-blue-500 hover:text-sky-400 no-underline`)}>
```

Comment:
Please remove the `twMerge` function from file as it is not needed here.

Reply:
I have updated the PR according to the review